eox_tenant/tenant_wise/proxies.py

```diff
@@ -126,25 +126,35 @@ def get_value_for_org(cls, org, val_name, default=None):
         if cached_value:
             return cached_value
 
-        tenant_config = TenantConfig.objects.values_list("lms_configs")
-        microsite_config = Microsite.objects.values_list("values")  # pylint: disable=no-member
+        result = None
+        source = "lms_configs" if getattr(settings, "SERVICE_VARIANT", "lms") == "lms" else "studio_configs"
```

Comment:
We can leave this part out and we will come back to it in a PR only to make this access calls transparent to the internal location of the configs